.env

```diff
@@ -137,3 +137,6 @@ SAFE_ORCHESTRATOR_PRIVATE_KEY=
 # Light Client
 LIGHT_CLIENT_PROXY_CONTRACT_ADDRESS=
 APPROVED_PROVER_ADDRESS=
+
+# Plonk Verification Library Deployment with Defender
+PLONK_VERIFIER_SALT=3
```

Comment:
Just a suggestion - but I think we should not push `.env` to main as its generally not advised and rather we should have a `.env.example` which can be converted to `.env` locally when needed. The reason for this is that someone can by accident add a private key to `.env` file and push it and bots will extract all the assets from that key - this has happened with a lot of people in the industry. Let me know what your thoughts are on this!

Reply:
yes, agree. 
we probably version control `.env` somehow by accident in the past.

Reply:
Thanks for calling this out so we can revise this. Historically, the `.env` file wasn't used for secrets but for config. The openzeppelin related config/secrets have been removed and placed into a gitignored env file called `.env.contracts` with a `.env.contracts.example` file provided as the example. This also resolves the comment above https://github.com/EspressoSystems/espresso-sequencer/pull/1718/commits/0e7ab30d5eb555e36755b9d724c60f76071ddfcd